### PR TITLE
Bug/61 Application page issues were fixed

### PR DESCRIPTION
**Pull request by vsaravanakumar**

### CODE REVIEW

File: front/src/components/User/EditUserModal.tsx [hunk 1]
```diff
@@ -70,6 +72,15 @@ export default function EditUserModal({
     }
   }, [rowData]);
 
+  useEffect(() => {
+    console.log("unique", uniqueValidation);
```
Comment (esakkirajaravenantech) [inline front/src/components/User/EditUserModal.tsx:76]: Remove console.log

File: front/src/components/User/UserList.tsx [hunk 2]
```diff
@@ -76,15 +74,11 @@ const UserList = () => {
   const handleEditSave = async (editedData: RowData) => {
     if ("id" in editedData) {
       editedData.user_name = editedData.userName;
-      delete editedData.userName;
+      // delete editedData.userName;
```
Comment (esakkirajaravenantech) [inline front/src/components/User/UserList.tsx:77]: Remove commented line

**Review by esakkirajaravenantech — CHANGES_REQUESTED** [review]

File: front/src/components/User/EditUserModal.tsx (hunk 1 above)
Comment (vsaravanakumar) [reply front/src/components/User/EditUserModal.tsx]: console.log removed

File: front/src/components/User/UserList.tsx (hunk 2 above)
Comment (vsaravanakumar) [reply front/src/components/User/UserList.tsx]: commented lines were removed

**Review by esakkirajaravenantech — APPROVED** [review]

**Pull request merged**